doc/kuksa_use_cases.md

```diff
@@ -0,0 +1,214 @@
+# KUKSA Use Cases
+
+
+# Content
+- [KUKSA Use Cases](#kuksa-use-cases)
+- [Content](#content)
+- [Use cases:](#use-cases)
+  - [1. Kuksa client subscribes on changed sensor values or actuator current values of all signals](#1-kuksa-client-subscribes-on-changed-sensor-values-or-actuator-current-values-of-all-signals)
+  - [2. Kuksa client sets a target value for an actuator.](#2-kuksa-client-sets-a-target-value-for-an-actuator)
+  - [3. Kuksa client gets sensor values or actuator current values of all signals.](#3-kuksa-client-gets-sensor-values-or-actuator-current-values-of-all-signals)
+  - [4. As a provider, I want to update atomically on Databroker, all sensor values as fast as possible.](#4-as-a-provider-i-want-to-update-atomically-on-databroker-all-sensor-values-as-fast-as-possible)
+  - [5. As a provider, if I receive a target value update through a subscription to the databroker, I want to modify the target value on the CAN bus and update the current value to the databroker if the operation is successful.](#5-as-a-provider-if-i-receive-a-target-value-update-through-a-subscription-to-the-databroker-i-want-to-modify-the-target-value-on-the-can-bus-and-update-the-current-value-to-the-databroker-if-the-operation-is-successful)
+
+# Use cases:
+
+## 1. Kuksa client subscribes on changed sensor values or actuator current values of all signals
+
+**Primary Actor:** Kuksa client
+
+**Secondary Actors:** Databroker, Provider, CAN 
+
+**Priority**: High
+
+**Preconditions:**
+ * Provider open CAN Bus connection and start receiving signal values.
```

Comment:
What do you think about abstracting this as "vehicle bus" or "vehicle network" instead of CAN?